.env_docker.example

```diff
@@ -28,3 +28,5 @@ WEBUI_SCRIPT=webui.py
 # Pass cli arguments to webui.py e.g:
 # WEBUI_ARGS=--optimized --extra-models-cpu --gpu=1 --esrgan-gpu=1 --gfpgan-gpu=1
 WEBUI_ARGS=
+
+STREAMLIT_SERVER_HEADLESS=true
```

Comment:
This wasn't needed in my testing on runpod.

Reply:
I was actually thinking about that when I noticed it got changed back. Without it, as of Streamlit 1.13, I get

```
sd-webui  |   👋 Welcome to Streamlit!
sd-webui  |
sd-webui  |   If you're one of our development partners or you're interested in getting
sd-webui  |   personal technical support or Streamlit updates, please enter your email
sd-webui  |   address below. Otherwise, you may leave the field blank.
sd-webui  |
sd-webui  |   Email:  entrypoint.sh: Process is ending. Relaunching in 0.5s...
```

Reply:
Ah yes i ran into this issue with runpod 

https://python.tutorialink.com/streamlit-showing-me-welcome-to-streamlit-message-when-executing-it-with-docker/

```
[general]
email=""
```
in `~/.streamlit/credentials.toml` is the fix